### ISSUE TITLE
CapturePreviewAsync exception

### ISSUE THREAD

**Issue by Stumec** [MicrosoftEdge/WebView2Feedback#2087]
**Description**
System.InvalidOperationException : 'CoreWebView2 members cannot be accessed after the WebView2 control is disposed
But WebView2 is not disposed

**Version**
SDK: 
Microsoft.Web.WebView2 1.0.1133-prerelease
Microsoft.Web.WebView2.DevToolsProtocolExtension 1.0.824
Framework: WinForms
OS:  Win10

**Repro Steps**
private async void CopieWebBitmap()
{
            MemoryStream ms = new MemoryStream();            
            await webBrowser1.CoreWebView2.CapturePreviewAsync(CoreWebView2CapturePreviewImageFormat.Jpeg, ms);            
            Global.xbrowserImage = new Bitmap(ms);
            ms.Close();
            ms.Dispose();
}

private void webBrowser1_CoreWebView2InitializationCompleted(object sender, Microsoft.Web.WebView2.Core.CoreWebView2InitializationCompletedEventArgs e)
        {
            CopieWebBitmap();         
        }

**Screenshots**

**Additional context**

**Comment by champnic** [comment]
@Stumec Thanks for the bug report! Does this happen on the `CapturePreviewAsync` line or somewhere else in the provided code? I wonder if it's related to being called in `InitializationCompleted`... What happens if you move that call somewhere else, like in response to a button click or `NavigationCompleted`?

**Comment by Stumec** [comment]
Yes on the CapturePreviewAsync 
It is good in response to a button click, exception in NavigationCompleted

**Comment by champnic** [comment]
Does you code keep the WebView2 alive while the call to `CapturePreviewAsync` is being made?

**Comment by Stumec** [comment]
Yes, the call is in the callback InitializationCompleted so WebView2 is alive

**Comment by champnic** [comment]
`CopieWebBitmap` is an async call that isn't being awaited, so the InitializationCompleted event is returning right away. If you had code that quickly spun up a WebView2 and then shut it down without waiting for `CopieWebBitmap` to complete, then that might be why you are seeing it. Do you have such code, or is the WebView2 control created and then kept around for a long time/past the screenshot capture?

**Comment by Stumec** [comment]
the WebView2 control persisted for long after screenshoot

**Comment by champnic** [comment]
Thanks for the info! I've added this bug to our backlog.

Is the NavigationCompleted exception the same thing?
